serverless-test-project/serverless.yml

```diff
@@ -12,7 +12,16 @@ custom:
     alarms:
       Lambda:
         Invocations:
-          enabled: true
+          enabled: false
```

Comment:
Why are we disabling these alarms for all Lambda functions?

Reply:
I am solving it. Alarms disabled for only the **eventsRule** lambda function now.